### ISSUE TITLE
[Issue]: Cloche crash.

### ISSUE THREAD

**Issue by phat-red** [The-Nexus-Project/Limitless-8#21]
### What happened?

game crashed while opening immersive engineering cloche with sugarcane

### Pack Version

Version: 1.21.1

### To Reproduce

1.open garden cloche with sugarcane

### Are you using Iris? (For shaders).

- [ ] Yes. I am using Iris.
- [X] No. My instance is unmodified, stock install.

### Are you on a Server?

- [ ] Yes.
- [X] No.

### whole latest.log output & crash report.

```shell
The game crashed whilst rendering block entity
Error: java.lang.IllegalArgumentException: The class blusunrize.immersiveengineering.client.utils.TransformingVertexBuilder does not implement interface VertexBufferWriter, which is required for compatibility with Sodium (see: https://github.com/CaffeineMC/sodium-fabric/issues/1620)
```

**Comment by Gbergz** [comment]
This is a known issue with Sodium currently, a fix is on the way from there.

**Comment by phat-red** [comment]
Thank you for your answer. I cant join the World without crash, what can i do for now?Am 27.10.24, 10:52 schrieb Gbergz ***@***.***>:

This is a known issue with Sodium currently, a fix is on the way from there.

**Comment by Gbergz** [comment]
you could temporarily remove sodium and its addons then remove the cloche and add sodium + addons back.

**Comment by Gbergz** [comment]
Will be fixed next update.

**Comment by phat-red** [comment]
Thank you very much, sirAm 31.10.24, 13:34 schrieb Gbergz ***@***.***>:

Will be fixed next update.

**Comment by Gbergz** [comment]
Fixed in 0.25.0, another crash with blueprints will be fixed in upcoming 0.26.0.
